x-pack/plugins/security_solution/server/endpoint/routes/metadata/metadata.test.ts

```diff
@@ -337,7 +339,7 @@ describe('test endpoint route', () => {
         });
         expect(mockResponse.notFound).toBeCalled();
         const message = mockResponse.notFound.mock.calls[0][0]?.body;
-        expect(message).toEqual('Endpoint Not Found');
+        expect(message).toBeInstanceOf(EndpointHostNotFoundError);
```

Comment:
FYI: I adjusted these tests (for the Endpoint Details) for the time being. but my goal is to remove most of them and move them to test the service. The testing of the API route handle will be very light (just ensuring that it passes data on to the service)